aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -443,7 +443,25 @@ Configure the following values for the client:
 * `serverTimeoutInMilliseconds`: The server timeout in milliseconds. If this timeout elapses without receiving any messages from the server, the connection is terminated with an error. The default timeout value is 30 seconds. The server timeout should be at least double the value assigned to the Keep-Alive interval (`keepAliveIntervalInMilliseconds`).
 * `keepAliveIntervalInMilliseconds`: Default interval at which to ping the server. This setting allows the server to detect hard disconnects, such as when a client unplugs their computer from the network. The ping occurs at most as often as the server pings. If the server pings every five seconds, assigning a value lower than `5000` (5 seconds) pings every five seconds. The default value is 15 seconds. The Keep-Alive interval should be less than or equal to half the value assigned to the server timeout (`serverTimeoutInMilliseconds`).
```

Comment:
Let's rewrite this section according to the current design. For TypeScript client new design is
```diff
export class HubConnectionBuilder
{
+    public withServerTimeout(milliseconds: number): HubConnectionBuilder;
+    public withKeepAliveInterval(milliseconds: number): HubConnectionBuilder;
}
```
Let's rephrase these portions of text where `serverTimeoutInMilliseconds` and `keepAliveIntervalInMilliseconds` are described. Replace with description for `withServerTimeout` and `withKeepAliveInterval` methods and keep the description of what  `serverTimeoutInMilliseconds` and `keepAliveIntervalInMilliseconds` actually are. Also let's add
`withServerTimeout` and `withKeepAliveInterval` methods configure `HubConnectionBuilder` and return the same instance of `HubConnectionBuilder` so they can be chained.

Reply:
Ignore previous comment